docs/source/includes/getting_started.md

```diff
@@ -8,6 +8,7 @@ Here you will see what you need to do to run the backend application.
 2. Conda: <https://www.anaconda.com/download>
 
 ## Initialising this repository
+
 Gentle reminder: you are currently in the `backend` repository, not the `CHORAS` parent repository. The instructions below therefore refer to the `backend` repository and neither involve the `CHORAS` nor `frontend` repositories.
 
 1. Clone this repository to a location of your choice. <!--If you have issues with cloning this repository (and its submodules), you can download the zipped repository via the releases page of this repository: <https://github.com/choras-org/backend/releases> -->
```

Comment:
```suggestion
1. Clone this repository to a location of your choice.
```
Since this can always be restored from the git history I'd remove the commented out section.